cli/module_graph.rs

```diff
@@ -610,7 +611,7 @@ pub struct BundleOptions {
 }
 
 #[derive(Debug, Default)]
-pub struct CheckOptions {
+pub struct CheckOptions<'a> {
```

Comment:
It seems like we are really going out of our way to avoid some cloning, which makes for some really ugly code.  I would much rather prefer we not throw pointer lifetimes everywhere and have to call stuff with `Cow::borrow`.

Reply:
`Cow` was the least obtrusive way I knew of to get it to satisfy `Default`, which was the only problem with a simple reference. I've changed it to just clone the the sets instead.

Reply:
😅 Turns out no cloning is necessary anyway, there's only one consumption per branch. My mistake.